src/systems/log/LogPlayback.cc

```diff
@@ -510,9 +510,9 @@ void LogPlayback::Update(const UpdateInfo &_info, EntityComponentManager &_ecm)
   {
     auto msgType = iter->Type();
 
-    // Support ignition.msgs for backwards compatibility, don't remove on tock
+    // Support gz.msgs for backwards compatibility, don't remove on tock
```

Comment:
I think it worth keeping backward compatibility here.

Reply:
Added back in [d3b2dd1](https://github.com/gazebosim/gz-sim/pull/2055/commits/d3b2dd1ac8f5321bb958e3a73e7de18496cbccce)